mobility_pipeline/lib/make_matrix.py

```diff
@@ -36,6 +36,17 @@
 import numpy as np  # type: ignore
 import pandas as pd  # type: ignore
 
+from typing import List, Union, cast
+from mypy_extensions import TypedDict
+import numpy as np  # type: ignore
+from shapely.geometry import Polygon, MultiPolygon 
```

Comment:
Also need to tell mypy to ignore this one
```suggestion
from shapely.geometry import Polygon, MultiPolygon  # type: ignore
```